### DISCUSSION TITLE
Some quick feedback on what I like and do not like about this program.

### DISCUSSION THREAD

**Discussion by Mirakelpung** [keiwando/toonsquid#596] [Ideas]
I've been animating regularly for maybe around 2 years now, and my main choice of tool for animating is standard Procreate on ipad.
Procreate is very comfortable for me to draw in, but it lacking rudimentary things like timeline and audio imports make this program less than ideal for someone who wants to make stories.

So I've tried out Toonsquid and in some regards its everything I'd want Procreate to be, it has a very streamlined UI and not a million menus to find the things you need, it has a timeline that works very well, and it has audio imports.

But my biggest problems with Toonsquid so far are mostly related to comfort of use, it simply does not feel as good as Procreate to draw in.
1. I am maybe an outlier here, but I mainly draw with my ipad held vertically. Even the times I do use a wider canvas, I like flipping my ipad around a lot to get the best angle for drawing nice lines. In procreate the UI will shift between horizontal and vertical when I flip it, but in Toonsquid it always remains horizontal. I really wish this was a feature you would implement.
2. There is no hover preview for the size of your brush. I think I've heard some users say this is an issue only on ipad and not on other platforms, but irregardless that is an issue for me. It makes it kind of hard especially when you need to make very precise lines or use the eraser for small details.
3. Something about the brush stabilizer just feels off and I just can't get it to make lines that aren't jittery or that changes its shape once I let go of the pencil. I'm still just an amateur artist for the most part, so I don't know enough about brushes to pinpoint exactly what is causing this. But I know that when I use the slider to stabilize lines it will either always make them very jittery, or it will simply not follow my pencil correctly.

Sorry if some of this feedback is pretty vague but I just wanted to give my 2 cents. This program feels like it has a lot of potential to overtake Procreate as my favorite tool for animation, but right now it lacks certain things to make this program feel just right for me. I've tried Procreate Dreams as well and I can safely say that I like Toonsquid a lot better because how the hell does someone create an art program without the lasso tool that's been a basic feature in art programs since mspaint on windows 98?

So anyway, good luck on your development of this program!

**Comment by Mirakelpung** [comment]
https://github.com/keiwando/toonsquid/assets/142930004/76163841-2f3f-4bff-84d1-7eb4ddabdd27

Right now I have mainly been using Toonsquid for importing short animations I've made in Procreate, and then adding sound to them. This is one of the ways Toonsquid has been useful to me, but this method of working in procreate and then porting it to Toonsquid still has some major issues when I want to make longer animations or sync animation with sound.

**Reply by inTIMidation297** [reply]
Nothing about ToonSquid but I love the little animation you made. Shsnshsnhs

**Comment by Mirakelpung** [comment]
https://github.com/keiwando/toonsquid/assets/142930004/4dbfedc6-b888-43f2-b2c1-60272ccd6fef

This is the longest animation I have made so far using Procreate and Toonsquid together (though I've only gotten halfway). It's with this animation I ran into my biggest issues with Toonsquid. I was hoping to draw the keyframes in Procreate, and then animate the tweens, do the coloring, sync movement and lips to the music, and ink in Toonsquid. However I find that I simply can't draw very well in toonsquid. I can't gauge the exact size of my brushes when I am drawing, I can't make clean inkwork because the brushes either move erratically or jitter, I can't clean up the lines with the eraser because the eraser size is invisible and it moves erratically, and all while doing this I am constantly battling the UI not being where I want it to be.

I really don't mean to shit on your creation, for a one man job Toonsquid still puts the Procreate dev team to shame. But I really do hope this program can become the competitor it deserves to be.

**Reply by keiwando** [reply]
No worries, I appreciate everybody who takes the time to provide feedback, especially in such detail. Thank you!

**Comment by Mirakelpung** [comment]
Hey, glad to see you got my feedback. You're doing a lot of good with this program.
